Gruntfile.js

```diff
@@ -3,78 +3,49 @@ module.exports = function (grunt) { // eslint-disable-line
 
 	var pkg = grunt.file.readJSON("package.json")
 	var currentYear = grunt.template.today("yyyy")
+
 	var inputFolder = "./docs"
-	var tempFolder = "./temp"
-	var archiveFolder = "./archive"
+	var layoutFolder = inputFolder + "/layout"
 	var outputFolder = "../mithril"
+	var archiveFolder = "./archive"
+	var currentVersionArchiveFolder = archiveFolder + "/v" + pkg.version
 
 	var guide = [
```

Comment:
I've removed `refactoring` which doesn't have a corresponding page. The rest is just re-ordering to match the navigation on the site (just for consistency, makes no practical difference). Should've split it over two commits to make that clearer.